src/test-utils.c

```diff
@@ -5,6 +5,11 @@
 //  Licensed under the "BSD-2-Clause Plus Patent License"
 //
 
+// Arduino has two problems with the tests:
+// 1. The test-utils are duplicated w/ other Blockchain Commons packages.
+// 2. The main routine conflicts w/ the actual main.
+#if !defined(ARDUINO)
```

Comment:
If you don't want the test code in the Arduino build, isn't there a better way of not including it that doesn't require modifying the source file?

Reply:
In the arduino world everything in the `src` directory gets automatically snarfed up and put in the library.

Could the tests live in an adjacent directory?